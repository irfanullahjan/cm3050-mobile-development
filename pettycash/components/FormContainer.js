import { StyleSheet, View } from "react-native";

export function FormContainer({ children }) {
  return <View style={styles.container}>{children}</View>;
}

const styles = StyleSheet.create({
  container: {
    padding: 15,
  },
});
