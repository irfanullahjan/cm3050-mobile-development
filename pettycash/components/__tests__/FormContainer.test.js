import { render } from "@testing-library/react-native";
import React from "react";

import { FormContainer } from "../FormContainer";

describe("FormContainer", () => {
  it("renders correctly", () => {
    const tree = render(<FormContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
