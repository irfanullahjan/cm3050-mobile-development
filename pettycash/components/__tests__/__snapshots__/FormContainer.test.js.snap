// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormContainer renders correctly 1`] = `
<View
  style={
    Object {
      "padding": 15,
    }
  }
/>
`;
